spec/common/providers/https.spec.ts

```diff
@@ -446,10 +476,11 @@ describe('onCallHandler', () => {
   });
 
   it('should reject bad AppCheck token', async () => {
+    const projectId = appsNamespace().admin.options.projectId;
+    const appId = '1:65211879909:web:3ae38ef1cdcb2e01fe5f0c';
```

Comment:
Is this fake/should we be exposing this here?

Reply:
Though this is vaguely PII it doesn't come with any security privileges. Think of this as hard-coding the name of your test project in code. We probably already did so with our mock credentials as well.

Reply:
For safe measures, I made the `appId` obviously fake.